core-common/src/main/java/org/zalando/nakadi/service/AvroSchema.java

```diff
@@ -24,12 +24,11 @@
 @Service
 public class AvroSchema {
 
-    public static final byte METADATA_VERSION = 0;
-    public static final Comparator<String> SCHEMA_VERSION_COMPARATOR = Comparator.comparingInt(Integer::parseInt);
+    public static final String METADATA_KEY = "_metadata";
```

Comment:
`_` is that indication of the internal_smth?

Reply:
to avoid user event type names `metadata`

Reply:
sort of. it's a temp. solution anyway and hopefully will be replaced with the proper registry